### PR TITLE
Document common processes

### PR DESCRIPTION
**Pull request by JohnFarrellDev**
## Link to JIRA ticket

No ticket.

### *What is the problem we're trying to solve*

- Document process for debugging locally (commenting out of sccvdbcontext)
- Document how we handle request validation
- Add Chris and Jack as active contributors 😀

#### _Checklist_

- [x] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x] Added tests to cover all new production code
- [x] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +138,27 @@ This will allow you to run the tests as normal in your IDE.
     - If we later make breaking changes to the table in use we rename the backup table to make it our `active` version of the table
     - Manually apply schema changes to the table we are interested in
 
+### Local Debugging
+
+- Non Windows users will have to deactivate the code running in [SccvDbContext](https://github.com/LBHackney-IT/social-care-case-viewer-api/blob/master/SocialCareCaseViewerApi/V1/Infrastructure/SccvDbContext.cs)
+- Comment out everything inside the `SccvDbContext()` function
+- Now when you run the application locally and make requests to localhost you should no longer get an error of type `OpenSslCryptographicException`
```
[the remark this answers is not in the document]
Comment (JohnFarrellDev) [reply README.md:145]: It's a great idea and something I can look into

File: README.md [hunk 2]
```diff
@@ -136,6 +138,27 @@ This will allow you to run the tests as normal in your IDE.
...
+
+### Validating Requests
+
+We are looking to use Fluent Validation to validate incoming network requests that contain a request body. The reason is that Fluent Validation comes with many common required validations out of the box but unlike data attributes it is easy to create our own custom validations and error handling.
+
+Some of our old code is still using data attributes but anything going forward should look to use the Fluent Validation library.
+
+Fluent Validation is a highly used library with a large community and well written [documentation](https://docs.fluentvalidation.net/en/latest/)
```
[the remark this answers is not in the document]
Comment (JohnFarrellDev) [reply README.md:153]: I'm all for it if everyone is happy to agree on Fluent Validation.

**Comment by JohnFarrellDev** [comment]
> Not sure if this is related, but I still get `System.Security.Cryptography.CryptographicException: Access is denied.` when I run the tests after commenting code from the `SccvDbContext` class.

We should definitely look into this, solution needs to work for everyone so that everyone can run this locally and debug more efficiently.

**Comment by Renny4Real** [comment]
> > Not sure if this is related, but I still get `System.Security.Cryptography.CryptographicException: Access is denied.` when I run the tests after commenting code from the `SccvDbContext` class.
> 
> We should definitely look into this, solution needs to work for everyone so that everyone can run this locally and debug more efficiently.

I think the tests work for windows but not for Macs but I was able to get the tests passing with a work around solution: 
Commenting out the `localTrustStore` variable and its usages in the [MongoDbTestContext.cs](https://github.com/LBHackney-IT/social-care-case-viewer-api/blob/f90d0e9c5af21590122971f1859dba06039764f9/SocialCareCaseViewerApi.Tests/MongoDbTestContext.cs) file. -> `line 18` & `lines 30 - 43`

But agree that we should try and find a solution for this that works for all.

**Pull request closed without merging**